tests/integration/modules/test_git.py

```diff
@@ -931,10 +931,16 @@ def test_worktree_add_rm(self):
             worktree_add_prefix = 'Preparing '
         else:
             worktree_add_prefix = 'Enter '
+
         worktree_path = tempfile.mkdtemp(dir=integration.TMP)
         worktree_basename = os.path.basename(worktree_path)
         worktree_path2 = tempfile.mkdtemp(dir=integration.TMP)
-        worktree_basename2 = os.path.basename(worktree_path2)
+
+        # Even though this is Windows, git commands return a unix style path
+        if salt.utils.is_windows():
+            worktree_path = worktree_path.replace('\\', '/')
+            worktree_path = worktree_path.replace('\\', '/')
```

Comment:
1. This was a typo on my part, should be `worktree_path2`
2. The output of Git returns Unix style os.sep on Windows

I will fix #1